problems/strings.js

```diff
@@ -3,23 +3,29 @@
  * @param {string} - str
  * @returns {string} - first char
  */
-function firstChar() {}
+function firstChar(str) {
+  return str.slice(0, 1);
```

Comment:
You can also return the first character with str[0] instead of using .slice()

Reply:
oooh. Thank you!